compose/start.sh

```diff
@@ -89,7 +89,7 @@ do
     elif [[ "${i}" == "-c" ]]; then
         debug="1"
         compose="automation-dataset-collection.yml"
-        workers=`ps auwwx | grep $USER | grep python3 | grep start_worker | awk '{print $2}'`
+        workers=`ps auwwx | grep $USER | grep -i python | grep start_worker | awk '{print $2}'`
```

Comment:
python or python3 for specificity?

Reply:
It doesn't show up as python3 on mac, just Python.